src/main.rs

```diff
@@ -385,7 +385,6 @@ fn main() {
 
         info!("Shutting down.");
         let shutdown = topology.stop();
-        metrics_trigger.cancel();
```

Comment:
@lukesteensen I can't find this var around, I guess it was removed?

Reply:
Yes. Sad we didn't notice before now :disappointed: